pylint/test/test_functional.py

```diff
@@ -231,6 +232,13 @@ def __init__(self, test_file):
         self._linter.disable('I')
         try:
             self._linter.read_config_file(test_file.option_file)
+            config_parser = self._linter.cfgfile_parser
+
+            if config_parser.has_option('MASTER', 'load-plugins'):
+                plugins = utils._splitstrip(
+                    config_parser.get('MASTER', 'load-plugins'))
+                self._linter.load_plugin_modules(plugins)
```

Comment:
I just realized that my plugin is never unregistered after a test is run.  Will have to rethink this.